stripe-ui-core/api/stripe-ui-core.api

```diff
@@ -246,6 +246,17 @@ public final class com/stripe/android/uicore/elements/TextFieldUIKt {
 	public static final fun TextFieldSection-vbMXUkU (Landroidx/compose/ui/Modifier;Lcom/stripe/android/uicore/elements/TextFieldController;IZZLjava/lang/Integer;Lkotlin/jvm/functions/Function1;Landroidx/compose/runtime/Composer;II)V
 }
 
+public final class com/stripe/android/uicore/elements/bottomsheet/ComposableSingletons$StripeBottomSheetLayoutKt {
```

Comment:
Can we get these out of here with a file:RestrictTo annotation?

Reply:
Can’t get rid of these, unfortunately.